stable/metrics-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.1
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.5.1
+version: 2.5.2
```

Comment:
Can you bump minor version?

Reply:
👍

Reply:
Actually, why would we update the minor version? Did you mean the patch version? The current version of the chart is already bumped appropriately. If you disagree, can you please elaborate?

Reply:
@olemarkus I'm not seeing any merge conflicts or otherwise and am confused by the above, please advise.

Reply:
Any new feature should be a minor increase, as @olemarkus suggested. You added a backwards compatible new option `serviceLabels` that didn't exist before and according to semver that warrants a minor bump.